### ISSUE TITLE
[Adaptive Cards - iOS - MediaInColumnSet.json]: Elements under video player are not accessible through the swipe gesture.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8436]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
Launch Adaptive Cards iOS application.
Home screen will appear.
Activate Voice over
Navigate to "V1.1" control and select it.
Navigate to 'Test' control and activate it.
Navigate to 'MediaInColumnSet.json' and activate it.
Card will appear.
Navigate to the 'Cat Image' button and double tap to activate it.
Now, perform right swipe and observe the issue.

**Actual Result:**
While navigating the 'MediaInColumnSet.Json' screen, after activating the video button, on performing the right swipe elements under player is not accessible through the swipe gesture.

**Expected Result:**
Elements under the player should be accessible through the swipe gesture.

**Observation:**
This issue is working fine with keyboard.

**User Experience:**
It will impact the screen reader user if the element under the player is not accessible as they will not be able to perform desired action.

WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/229017513-6780bc17-2369-40e1-a39e-837296df6568.mov

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
closing this issue, the app wasn't built correctly perhaps, only cards under Scenario should be tested, not under Elements. I will make note of this and fix it.

**Comment by vagpt** [comment]
Adding A11yTTValidated tag as per the above comment.
